llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -2343,8 +2345,9 @@ class LLVM_ABI_FOR_TEST VPWidenPHIRecipe : public VPSingleDefRecipe,
   }
 
   VPWidenPHIRecipe *clone() override {
-    auto *C = new VPWidenPHIRecipe(cast<PHINode>(getUnderlyingValue()),
-                                   getOperand(0), getDebugLoc(), Name);
+    auto *C =
+        new VPWidenPHIRecipe(cast_if_present<PHINode>(getUnderlyingValue()),
```

Comment:
When I change the cast to `cast<PHINode>(getUnderlyingValue())` none of the tests change. Is this still necessary?